packages/adblocker-electron/adblocker.ts

```diff
@@ -110,7 +110,6 @@ export class BlockingContext {
     if (this.blocker.config.loadCosmeticFilters === true) {
       this.session.setPreloads(this.session.getPreloads().filter((p) => p !== PRELOAD_PATH));
       ipcMain.removeListener('get-cosmetic-filters', this.onGetCosmeticFilters);
-      ipcMain.removeListener('is-mutation-observer-enabled', this.onIsMutationObserverEnabled);
```

Comment:
Can we maybe add a comment here so that we know we have to do something about it eventually?

Reply:
In any case I'm gonna merge and publish this and we can revisit soon hopefully.